trading/models.py

```diff
@@ -0,0 +1,65 @@
+from django.core.validators import MinValueValidator
+from django.db import models
+from django.contrib.auth.models import User
+from item.models import Item
+from trading.enums import OrderType
+
+
+class Offer(models.Model):
+    """Request to buy or sell specific stocks"""
+    user = models.ForeignKey(
+        User,
+        on_delete=models.CASCADE,
+        related_name="offers"
+    )
+    item = models.ForeignKey(
+        Item,
+        on_delete=models.CASCADE,
+        related_name="offer"
```

Comment:
Your Item model will have a set of offers, so i think you need to use plural `offers`. Like above in User